pkg/agent/cache/manager.go

```diff
@@ -60,12 +60,15 @@ type Manager interface {
 	GetTeamByID(id string) *definitions.PlatformTeam
 	GetDefaultTeam() *definitions.PlatformTeam
 
+	//Access Request cache related methods
+	GetAccessRequestCache() cache.Cache
```

Comment:
May be add methods to manage(get, add, delete) the access request cache and use that instead.

Reply:
This is also not needed until the TAs start reporting usages against access request details, should we just remove for now?